tests/integration/api/queries/ordersByAccountId/ordersByAccountId.test.js

```diff
@@ -0,0 +1,233 @@
+  shopId,
+  payments: {
+    _id: (iterator) => (iterator + 33).toString(),
+    data: {
+      fullName: "Reaction Commerce",
+      gqlType: "ExampleIOUPaymentData"
+    },
+    displayName: "IOU from Reaction Commerce",
+    method: "credit",
+    mode: "authorize",
+    name: "iou_example",
+    paymentPluginName: "example-paymentmethod",
+    processor: "Example",
+    riskLevel: "normal",
+    shopId,
+    status: "created",
+    transactionId: "s9atGoLbagKvQ3pJc"
+  },
+  workflow: {
+    status: "new"
+  }
+});
+
+const canceledOrders = Factory.Order.makeMany(3, {
+  _id: (iterator) => (iterator + 600).toString(),
+  accountId: mockAccount._id,
+  referenceId: (iterator) => (iterator + 323).toString(),
+  shopId,
+  payments: {
+    _id: (iterator) => (iterator + 11).toString(),
+    data: {
+      fullName: "Reaction Commerce",
+      gqlType: "ExampleIOUPaymentData"
+    },
+    displayName: "IOU from Reaction Commerce",
+    method: "credit",
+    mode: "authorize",
+    name: "iou_example",
+    paymentPluginName: "example-paymentmethod",
+    processor: "Example",
+    riskLevel: "normal",
+    shopId,
+    status: "created",
+    transactionId: "s9atGoLbagKvQ3pJc"
+  },
+  workflow: {
+    status: "coreOrderWorkflow/canceled"
+  }
+});
+let testApp;
+let query;
+
+beforeAll(async () => {
+  testApp = new TestApp();
+  await testApp.start();
+  query = testApp.query(ordersByAccountIdQuery);
+  await testApp.createUserAndAccount(mockAccount);
+  await testApp.insertPrimaryShop({ _id: shopId, name: "Shop Name" });
+});
+
+beforeEach(async () => {
+  await testApp.clearLoggedInUser();
+  await testApp.collections.Orders.deleteMany({});
+});
+
+afterAll(async () => {
+  await testApp.collections.Orders.deleteMany({});
+  await testApp.collections.Accounts.deleteMany({});
+  await testApp.collections.Shops.deleteMany({});
+  await testApp.stop();
+});
+
+test("get 1 order by account ID, without shopID or order status", async () => {
+  await testApp.collections.Orders.insertOne(order);
+  await testApp.setLoggedInUser(mockAccount);
+  const result = await query({ accountId: opaqueAccountId });
+  expect(result).toBeTruthy();
+  expect(result.ordersByAccountId.nodes[0].account._id).toBe(opaqueAccountId);
+});
+
+test("get multiple orders by account ID, without shopID or order status", async () => {
+  await Promise.all(orders.map((mockOrder) => testApp.collections.Orders.insertOne(mockOrder)));
+  await testApp.setLoggedInUser(mockAccount);
+  const result = await query({ accountId: opaqueAccountId });
+  expect(result).toBeTruthy();
+  expect(result.ordersByAccountId.nodes[0].account._id).toBe(opaqueAccountId);
+  expect(result.ordersByAccountId.nodes[1].account._id).toBe(opaqueAccountId);
+  expect(result.ordersByAccountId.nodes[3].account._id).toBe(opaqueAccountId);
+  expect(result.ordersByAccountId.nodes[4].account._id).toBe(opaqueAccountId);
+  expect(result.ordersByAccountId.totalCount).toBe(10);
```

Comment:
This would be a better test if you also inserted one order that belongs to a different account, and verify that only the 10 for this account come back.